tools/turbinia@.service

```diff
@@ -22,30 +22,23 @@
 # sudo systemctl stop turbinia@psqworker
 # sudo systemctl daemon-reload
 
-
 [Unit]
 Description=Turbinia %i Daemon
 Documentation=https://github.com/google/turbinia
 Before=multi-user.target
 After=network.target
 After=remote-fs.target
 
-
 [Service]
 Type=simple
 PIDFile=/var/run/turbinia-%i.pid
 User=turbinia
 Group=turbinia
 Restart=always
 RestartSec=10
-ExecStartPre=+/bin/sh -c '/bin/systemctl set-environment GOOGLE_APPLICATION_CREDENTIALS=$(turbinia_config_parser.py GCE_SERVICE_ACCOUNT_KEYS_FILE)'
-ExecStartPre=+/bin/sh -c '/bin/systemctl set-environment TURBINIA_TMPDIR=$(turbinia_config_parser.py TMP_DIR)'
-ExecStart=/bin/sh -c '/usr/local/turbinia/turbiniactl -L $TURBINIA_TMPDIR/turbinia-%i.log -S -o $TURBINIA_TMPDIR %i 2>> $TURBINIA_TMPDIR/turbinia-%i.stdout.log'
-ExecStopPost=+/bin/sh -c 'systemctl unset-environment TURBINIA_TMPDIR'
-ExecStopPost=+/bin/sh -c 'systemctl unset-environment GOOGLE_APPLICATION_CREDENTIALS'
+ExecStart=/bin/sh -c '/usr/local/bin/turbiniactl %i'
```

Comment:
I still want to see this working with the local service account with permissions scoped to the instance before we remove these, ;)

Reply:
fair enough ;) I let you know